src/Microsoft.VisualStudio.Services.Agent/Terminal.cs

```diff
@@ -37,6 +39,25 @@ public override void Initialize(IHostContext hostContext)
         {
             base.Initialize(hostContext);
             Console.CancelKeyPress += Console_CancelKeyPress;
+
+            var terminalEncoding = Encoding.UTF8;
```

Comment:
This should probably be behind a feature flag (off by default)

Reply:
The main goal is to achieve compatibility with on-prem solutions as well. I think it should work by default.